Makefile

```diff
@@ -49,8 +53,8 @@ endif
 all: manager
 
 # Run tests
-test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+test: generate fmt vet manifests $(GO_JUNIT_REPORT)
+	go test ./... -v 2>&1 | $(GO_JUNIT_REPORT) > $(REPORT_NAME).xml
```

Comment:
I'm wondering if we can make optional, generate xml only when mentioned(especially via CI scripts)